source/mongodb/crud-and-aggregation-apis.txt

```diff
@@ -599,20 +607,11 @@ App Services supports all :manual:`aggregation pipeline operators
 </reference/operator/aggregation/>` when you :ref:`run an aggregation pipeline
 <mongodb-aggregate>` in the :ref:`system user
 <system-functions>` context. App Services supports all pipeline operators
-in an :ref:`application user <user-accounts>` context with the following
-exceptions:
-
-.. list-table::
-   :header-rows: 1
-   :widths: 25 10 10
+in an :ref:`application user <user-accounts>` context except for the following
+operators:
 
-   * - Operator
-     - User Context
-     - System Context
-
-   * - :manual:`$function </reference/operator/aggregation/function/>`
-     - No
-     - Yes
+- :manual:`$function </reference/operator/aggregation/function/>`
+- :manual:`$setIsSubset </reference/operator/aggregation/setIsSubset/>`
 
 .. _mongodb-crud-and-aggregation-apis-database-commands:
```

Comment:
did we confirm that `distinct` is fully unsupported?

Reply:
As per my note above, I did not add distinct because it is an aggregation command, not an operator.

Reply:
correct, but should it be added to the table below (the mongodb-crud-and-aggregation-apis-database-commands section)